snippets/base/util.py

```diff
@@ -96,3 +98,7 @@ def __call__(self, matchobj):
 
 def to_unix_time_seconds(dt):
     return int((dt - EPOCH).total_seconds())
+
+
+def get_sentinel_user():
+    return get_user_model().objects.get_or_create(username='deleted')[0]
```

Comment:
I read about this approach to allow user deleting without losing the related objects. But on second thought, I don't want to lose the user information entirely, so I'll just deactive the user instead of deleting. 

I changed this line to `models.PROTECT` to prevent user deletion when user has related objs.